manim/mobject/geometry.py

```diff
@@ -1424,11 +1489,16 @@ def construct(self):
     """
 
     def __init__(self, start_anchor, start_handle, end_handle, end_anchor, **kwargs):
-        VMobject.__init__(self, **kwargs)
-        self.set_points([start_anchor, start_handle, end_handle, end_anchor])
+        super().__init__(**kwargs)
+        if config.renderer == "opengl":
+            self.add_cubic_bezier_curve(
+                start_anchor, start_handle, end_handle, end_anchor
+            )
+        else:
+            self.set_points([start_anchor, start_handle, end_handle, end_anchor])
```

Comment:
Imo when methods other than manipulating the raw points exist, they should be used
```suggestion
        self.add_cubic_bezier_curve(
            start_anchor, start_handle, end_handle, end_anchor
        )
```

Reply:
Of course, makes sense -- I somehow assumed that `add_cubic_bezier_curve` was a OpenGLVMobject-specific thing.